.github/workflows/publish.yml

```diff
@@ -26,62 +26,60 @@ jobs:
 
       - name: Publish
         run: |-
-          cp .jvmopts-ghactions .jvmopts
-          sbt ci-release
-          CI_RELEASE=codegen/publishSigned CI_SNAPSHOT_RELEASE=codegen/publish sbt ++2.13.8\! ci-release
+          cp .jvmopts-ci .jvmopts
+          sbt +publish
```

Comment:
So from what I can tell the `+publish` is all that is needed here? I did a `+publishLocal` and at least from what I could tell it works properly.

Reply:
@jrudolph @raboof Maybe you might have some context here

Reply:
I don't have the details, but I seem to remember `+` would publish the same artifact(s) twice for some of the Scala versions, which Sonatype wouldn't accept. If we're now first publishing to an Apache repo instead of directly to Sonatype it's indeed very possible that this is no longer needed.

Also, 'publish' unlike 'ci-release' doesn't do signing etc, but that might not be a problem for the same reason.

Reply:
> I don't have the details, but I seem to remember `+` would publish the same artifact(s) twice for some of the Scala versions, which Sonatype wouldn't accept. If we're now first publishing to an Apache repo instead of directly to Sonatype it's indeed very possible that this is no longer needed.

Is this problem specific for this project or just in general?

> Also, 'publish' unlike 'ci-release' doesn't do signing etc, but that might not be a problem for the same reason.

Yes, this is just for snapshots which are currently not signed. Actually release is something we still need to decide upon later

Reply:
> Is this problem specific for this project or just in general? Afaik `+publish` in general should just work so I guess this project has set things up differently?

I think it had to do with the fact that in this project, some submodules are released for different Scala versions than others.

> Yes, this is just for snapshots which are currently not signed. Actually release is something we still need to decide upon later

:+1: